pkg/generate/generate.go

```diff
@@ -0,0 +1,114 @@
+package generate
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+
+	"moul.io/climan"
+)
+
+func Generate() *climan.Command {
+	var moduleRepo string
+	var outputDir string
+	var templateDir string
+
+	return &climan.Command{
+		Name:       "generate",
+		ShortHelp:  "Generate code, you must specify the path to your protoc bin in the environment variable PROTOC_BIN",
+		ShortUsage: "adapterkit generate [global flags] [flags] [args]",
+		FlagSetBuilder: func(fs *flag.FlagSet) {
+			fs.StringVar(&moduleRepo, "mod", "", "github repo where the module is located")
+			fs.StringVar(&outputDir, "out", ".", "output directory")
+			fs.StringVar(&templateDir, "tpl", "template", "template directory")
+		},
```

Comment:
```suggestion
        fs.StringVar(&protocBin, "protoc-bin", "protoc", "path to the 'protoc' binary)
		},
```

and you can configure climan to accept env vars.